common/src/main/java/org/conscrypt/OpenSSLCipher.java

```diff
@@ -1196,7 +1196,10 @@ protected AlgorithmParameters engineGetParameters() {
                 AlgorithmParameters params = AlgorithmParameters.getInstance("GCM");
                 params.init(spec);
                 return params;
-            } catch (NoSuchAlgorithmException | InvalidParameterSpecException e) {
+            } catch (NoSuchAlgorithmException e) {
+                // This may happen since Conscrypt doesn't provide this itself.
```

Comment:
We do actually provide this now, so NoSuchAlgorithmException should be impossible.